packages/components/src/components/gradient-background-two-point/gradient-background-two-point.tsx

```diff
@@ -2,27 +2,27 @@ import React, { CSSProperties } from 'react';
 import './gradient-background-two-point.scss';
 
 type TGradientBackgroundTwoPoint = {
-    blurRadius?: number;
+    blur_radius?: number;
     backgroundColor: CSSProperties['backgroundColor'];
```

Comment:
@sergei-deriv make the others snake_case as well.

Reply:
done